test/test_mathics_precedence.py

```diff
@@ -455,9 +455,8 @@
     "TildeEqual",
     "TildeFullEqual",
     "TildeTilde",
-    # In Mathics, the precedence of these operators is quite low.
-    #    "DirectedEdge",  # Mathics 128 , WMA 295
-    #    "UndirectedEdge", # Mathics 120, WMA 295
+    "DirectedEdge",
```

Comment:
The precedence order of these symbols is tested here. Of course, other tests can be added.

Reply:
This test is extremely opaque. (BTW the one for TestConstrained is also that way.)

I was hoping for something _operational_  or something a user might experience, rather than checking that the value in this table is less than another value. 

Setting operator precedence determines whether we have parenthesis or not. So in a chain of direct/directed operators, we should see in full form the grouping change, and when there are other operators around them, then either parenthesis or the way calls are nested change.  

I seem to recall that there were tests around that worked that way. This is testing the effect of the precedence values on the way parsing happens.

Reply:
The comment for this PR shows WMA code  that was run demonstrating _why_ the precedence needs to be changed.

I want the tests to reflect that by doing the same thing.  In general, if one observes that something is wrong because when I run something I get this, then fixing that should run the demonstration to show this is now addressed.

In the future, developers are more likely to encounter the pytest code rather than some PR commentary.  The PR commentary is not checked in a simple-minded and easy-to-understand way.

Reply:
OK, but the comments have the only purpose to converge to some criteria about what value must be assigned. Once these criteria can be clearly written, the comments can go out.  I started to do that in #105, over this.

Reply:
#105 is fine (with one small removal of slight vagueness). 

_This_ PR changes the operator precedence  numerical _value_ of `DirectedEdge` and `UndirectedEdge`. I am asking that a test be written to demonstrate why the operator precedence is needed and to check that the operational effect we want to achieve persists in the future.